OpenSim/Simulation/SimulationUtilities.h

```diff
@@ -94,6 +94,22 @@ void updatePre40KinematicsFilesFor40MotionType(const Model& model,
         const std::vector<std::string>& filePaths,
         std::string suffix="_updated");
 
+/** This function attempts to update the connectee path for any Socket anywhere
+ * in the model whose connectee path does not point to an existing component.
+ * The paths are updated by searching the model for a component with the
+ * correct name. For example, a connectee path like
+ * `../../some/invalid/path/to/foo` will be updated to `/bodyset/foo` if a Body
+ * named `foo` exists in the Model's BodySet.
+ *
+ * This function does not update the connectee paths for Inputs.
+ *
+ * This function is mainly for use with models loaded from files with XML
```

Comment:
"mainly for use with" doesn't really specify what I can/should do. Perhaps something like "This method is intended for use with models loaded from version-30516 XML files to bring them up to date with the 4.0 API". I would prefer to see a report after execution saying either "No Sockets needed updating; Model was not changed" or "x Sockets were updated".

Reply:
Done.